gateway/gateway.go

```diff
@@ -697,6 +699,12 @@ type pendingEventsRequestPayload struct {
 	JobRunID      string `json:"job_run_id"`
 }
 
+type pendingWHResponse struct {
```

Comment:
use same struct that is used in warehouse ?

Reply:
Done